sdk/documentintelligence/Azure.AI.DocumentIntelligence/samples/Sample_AnalyzePrebuiltDocument.md

```diff
@@ -0,0 +1,98 @@
+# Analyze with the prebuilt general document model
```

Comment:
Note: this sample fails with a 404. At the moment I'm trying to double check if this is a service bug or if they stopped supporting the prebuilt-document model. I'm thinking about replacing this sample with another one if I can't confirm it on time for the release.

Reply:
It's confirmed that this model has been deprecated. I'll update the sample to use `prebuilt-layout` instead. The contents of the code may change a bit.